media-plugins/vdr-devstatus/metadata.xml

```diff
@@ -1,8 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
-<pkgmetadata>
-  <maintainer type="project">
-    <email>vdr@gentoo.org</email>
-    <name>Gentoo VDR Project</name>
-  </maintainer>
+	<pkgmetadata>
+		<maintainer type="person">
+			<email>martin.dummer@gmx.net</email>
+			<name>Martin Dummer</name>
+		</maintainer>
+		<maintainer type="project">
+			<email>proxy-maint@gentoo.org</email>
+			<name>Proxy Maintainers</name>
+		</maintainer>
+		<maintainer type="project">
+			<email>vdr@gentoo.org</email>
+			<name>Gentoo VDR Project</name>
+		</maintainer>
+		<longdescription>
+			VDR Plugin to show the current status of dvb devices (tuned channes, recordings) on OSD
```

Comment:
This isn't any longer than the default ebuild description and could be removed :)

Reply:
ok. not on my "git blame" report...